Add content about installing project on IPv6 (#254)
theforeman/foreman-documentation: guides/common/assembly_preparing-environment-for-satellite-installation-ipv6.adoc

> [id="preparing-environment-for-{project}-installation-ipv6"]

Reviewer: Do you really need to mention the project in the ID? Isn't it better to avoid it when possible?

Author: That's a good point, it is probably better to avoid using project in the ID. However, in some cases there will be similar modules for Foreman and Smart Proxy. In this cases we need to specify this in the ID anyway. For example, in `proc_installing-the-satellite-server-packages.adoc` and `proc_installing-capsule-server-packages.adoc`.

Reviewer: Agreed, but I'd suggest to leave it out when you can. It can even avoid confusion. Here the user may be looking for a document to prepare the capsule but as I read this, it's more of an integral env preparation.

Reviewer 2: Another discussion we need to have. The obvious solution is to remove these from all IDs however this will likely break links in existing Satellite docs. Maybe we can add two IDs to those places with a comment that the old one is legacy for keeping links alive.

= Preparing your Environment for {Project} Installation in an IPv6 Network

> If you must provision hosts in IPv6 network, you can install {Project} in a IPv6 network and provision using UEFI HTTP Boot. Before installing {Project} in an IPv6 network, view the limitations that such scenario has and ensure that you meet the requirements.

Reviewer: I'd use an impersonal form.
```suggestion
To provision hosts in IPv6 networks, {Project} should be installed in an IPv6 network and UEFI HTTP Boot configured. Before installing {Project} in an IPv6 network, view the limitations on the scenario and ensure that the requirements are met.
```

Author: I applied your technical corrections here and fixed the style a bit. The problem with an impersonal form, is that it is harder to translate and to understand. For example, with passive voice you do not see who performs the action. So as per the IBM style guide, we use active voice as often as possible.
